tests/unit/nupic/algorithms/cla_classifier_test.py

```diff
@@ -28,7 +28,9 @@
 import unittest2 as unittest
 
 import numpy
+import tempfile
 
+from nupic.bindings.proto import ClaClassifier_capnp
```

Comment:
Never mind.. I forgot add this to `algorihtms.i`:

`#include <nupic/proto/ClaClassifier.capnp.h>`